admin-dev/themes/new-theme/js/header.js

```diff
@@ -24,13 +24,16 @@
  */
 import refreshNotifications from './notifications.js';
 
+const $ = window.$;
```

Comment:
Why sometimes `window.$` and sometimes `global.$`;

More you should be able to use `import $ from 'jquery';` with the right dependencies.

Reply:
I agree that it would be best to `import $ from 'jquery';`, but I believe jQuery is being inlined because of non compiled scripts on the page. Therefore, I fear that importing it as a module would include jQuery twice.

I don't really know if there's a difference between `global` and `window` when fetching global objects from document scripts.

Reply:
It's just to be more consistent, maybe use only `window.$` :smiley: